doc/mundo.txt

```diff
@@ -241,23 +253,32 @@ for a faster playback.
 Default: 60
 
 ------------------------------------------------------------------------------
-3.13 g:mundo_mirror_graph                                *mundo_mirror_graph*
+3.14 g:mundo_mirror_graph                                *mundo_mirror_graph*
 
 Set this to 0 to align the graph to the left; set to 1 to align to the right.
 
 Default: 1 (mirror graph)
 
 ------------------------------------------------------------------------------
-3.14 g:mundo_inline_undo                                 *mundo_inline_undo*
+3.15 g:mundo_inline_undo                                 *mundo_inline_undo*
 
 When enabled, a small one line diff is displayed to the right of the graph undo.
 Although not as detailed as a full diff provided in the preview window, it
 provides a quick summary of the diff w/o having to navigate.
 
 Default: 0 (no inline graph)
 
+-----------------------------------------------------------------------------
+3.16 g:mundo_enable_inline_delay                   *mundo_enable_inline_delay*
+
```

Comment:
As mentioned in comments above, this setting effectively does nothing. I don't see a benefit to having two different _delay settings either - I suggest we remove it.